tools/plugins/com.liferay.ide.server.ui/src/com/liferay/ide/server/ui/wizard/RemoteServerComposite.java

```diff
@@ -340,7 +340,15 @@ public void run()
                             else if( updateStatus.getSeverity() == IStatus.WARNING ||
                                 updateStatus.getSeverity() == IStatus.ERROR )
                             {
-                                wizard.setMessage( updateStatus.getMessage(), IMessageProvider.WARNING );
+                            	if( updateStatus.getMessage().contains("Your license key has expired") || 
+                            			updateStatus.getMessage().contains("Register Your Server or Application") )
```

Comment:
Lets make the check for the message a little more robust, check some other parts of the message as well to make sure it is that problem.  Also the mesage that we need to add to the wizard should be "Your license key has expired"